src/Nethermind/Nethermind.Monitoring/Config/IMetricsConfig.cs

```diff
@@ -18,17 +18,20 @@
 
 namespace Nethermind.Monitoring.Config
 {
-    [ConfigCategory(Description = "Configuration of the Prometheus + Grafana metrics publication. Documentation of the required setup is not yet ready (but the metrics do work and are used by the dev team)")]
+    [ConfigCategory(Description = "Configuration of the Prometheus metrics publication. Documentation of the required setup is not yet ready (but the metrics do work and are used by the dev team)")]
     public interface IMetricsConfig : IConfig
     {
-        [ConfigItem(Description = "If 'true' then the node publishes various metrics to Prometheus at the given interval.", DefaultValue = "false")]
-        bool Enabled { get; }
+        [ConfigItem(Description = "If set, the node exposes Prometheus metrics on the given port.", DefaultValue = "null")]
+        int? ExposePort { get; }
+
+        [ConfigItem(Description = "If 'true',the node publishes various metrics to Prometheus Pushgateway at given interval.", DefaultValue = "false")]
+        bool PushEnabled { get; }
```

Comment:
cannot change config names without redirecting old settings -> this would break installations for other users.
we can add alternative setting names that take precedence over others if set.